src/database/modele/Discussion.js

```diff
@@ -36,19 +38,38 @@ const DiscussionSchema = new mongoose.Schema({
         refParticipant: {
             type: Number,
             ref: 'Participant',
-            required: true
+            required: true,
+            //Validate will permit us to make some validation before a refParticipant is saved
+            //The function check if the id of the value passed exits in the DataBase if yes it will return true otherwise false
```

Comment:
```suggestion
            // Validate will permit us to make some validation before a refParticipant is saved
            // The function check if the id of the value passed exits in the DataBase if yes it will return true otherwise false
```